server/datastore/mysql/migrations/tables/20240925171917_AddScheduleAutomationsIndex.go

```diff
@@ -0,0 +1,25 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20240925171917, Down_20240925171917)
+}
+
+func Up_20240925171917(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+		CREATE INDEX idx_queries_schedule_automations ON queries (schedule_interval, automations_enabled)
```

Comment:
We don't care about the specific value of `schedule_interval` -- we just want to know if it zero or not? Maybe it is better to have a virtual stored column (`schedule_interval > 0`) and index/query on that?

Reply:
super intrigued about this.  I'll test

Reply:
didn't dig super deep into this since the existing query is already performant, but the query plan doesn't seem to want to use the new index on the virtual column.  Looking into this further, i'm not sure there will be much improvement since virtual columns results are generated on demand https://dev.mysql.com/doc/refman/8.0/en/create-table-generated-columns.html

Reply:
Sorry, I meant `generated stored` column, and not `virtual`

Reply:
This did show some improvement!  Latest commit adds a stored generated column `is_scheduled` on `scheduled_interval > 0`